docs/getting_started/openstack.md

```diff
@@ -218,6 +218,16 @@ spec:
       insecureSkipVerify: true
 ```
 
-## Next steps
+# Security of node bootstrap
+
+As of kOps 1.27, the node bootstrap process uses an instance's UUID as a plaintext password.
+Anything able to obtain this UUID and query port 3988 on the control plane will be
+able to impersonate the instance and join the cluster as that node.
+
+Furthermore, any instance in the account which has a `KopsInstanceGroup` tag
```

Comment:
this is not true. The server needs to be in same OpenStack network, at the same time it means that user needs to have admin access to OpenStack project in order to provision these servers/modify metadata in existing servers. If someone gets somehow admin access to OpenStack project, it is much more bigger issue than kops-controller auth. Basically this can be compared that someone will get access in AWS to provision ec2 instances / modify existing ones. Why we are not documenting that issue?

Reply:
also instance needs to be in nodes security group, the 3988 port is open only for that.